src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -135,20 +138,36 @@ public void autonomousPeriodic() {
 	 */
 	public void teleopPeriodic() {
 		driverstation.readInputs();
-		double speed = driverstation.getArcadeSpeed();
-		double turn = driverstation.getArcadeTurn();
-		
+		//TODO: Set Min_DRIVE_SPEED in Robot Map.
+		double MIN_DRIVE_SPEED = 0.1;
+		driverstation.readInputs();
+		double left = driverstation.getArcadeSpeed();
+		double right = driverstation.getArcadeTurn();
+		 		// -1* driverstation.getDriveJoystick().getJoystick()
+		LOGGER.info("left " + left + " right " + right) ;
```

Comment:
I don't think you want this left on all the time. Maybe drop it down to a trace level message.